samples/cext-trey-research-python/appManifest/trey-declarative-copilot.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://aka.ms/json-schemas/copilot-extensions/vNext/declarative-copilot.schema.json",
```

Comment:
Let's use the v1.0 schema for declarative agents

https://aka.ms/json-schemas/agent/declarative-agent/v1.0/schema.json

Reply:
The json link provided above is giving error and so used the below one 
https://aka.ms/json-schemas/copilot/declarative-agent/v1.0/schema.json